### PR TITLE
Update fhir version in pom.xml

### PR DESCRIPTION
**Pull request by lauravignoli**
This is necessary for the sync2 module to start

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -70,7 +70,7 @@
 		<uitestframeworkVersion>2.3.0</uitestframeworkVersion>
 		<legacyuiVersion>1.4.0</legacyuiVersion>
 		<reportingcompatibilityVersion>2.0.5</reportingcompatibilityVersion>
-		<fhirVersion>1.13.0</fhirVersion>
+		<fhirVersion>1.14.0-SNAPSHOT</fhirVersion>
```
Comment (kaweesi) [inline pom.xml]: we aren't using fhir yet except that its depended on by sync2, any reason why we are updating it?

Comment (lauravignoli) [reply pom.xml]: Because otherwise sync2 doesn't start? We actually need 1.13.1, so maybe that one makes more sense?

Comment (kaweesi) [reply pom.xml]: let us go with 1.13.1

Comment (lauravignoli) [reply pom.xml]: I will change it to 1.13.1

**Pull request merged**